client/src/components/BookingsDatePicker/useStyles.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prettier/prettier */
+import { makeStyles } from '@material-ui/core/styles';
+
+const red = "#FF0000";
```

Comment:
The red you picked doesn't seem to match the mock-up very well. Could we pick a lighter red?

Reply:
You're right. I update the theme primary color.